swagger_to_sdk_config.json

```diff
@@ -1,18 +1,17 @@
 {
-  "$schema": "https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/schemas/swagger_to_sdk_config.schema.json",
-  "meta": {
-    "autorest_options": {
-      "version": "V2",
-      "typescript": "",
-      "license-header": "MICROSOFT_MIT_NO_VERSION",
-      "sdkrel:typescript-sdks-folder": ".",
-      "use": "@microsoft.azure/autorest.typescript@4.7.0"
+  "advancedOptions": {
+    "createSdkPullRequests": true,
+    "generationCallMode": "one-per-config"
```

Comment:
Why do you prefer one-per-config? Is there any reason?

Reply:
If we use `one-for-all-configs` and there are multi RPs in a PR, it will generates multi PR for every RP. Then, the different RP all changes the pnpm-lock.json file and rush.json file. There will be in a mess.

Reply:
Oh yes that makes sense.